scss/_mixins.scss

```diff
@@ -73,27 +61,15 @@
     .h4,
     .h5,
     .h6 {
```

Comment:
May be able to replace lines 51 - 63 with a * , but not sure if that is too heavy handed.